### ISSUE TITLE
The Inshop Datasets accuracy

### ISSUE THREAD

**Issue by zcgit001** [euwern/proxynca_pp#10]
Hi, thanks for your code.
I used your code to run experiments on these four datasets, cub, cars, inshop and sop,  but the experimental results on the Inshop dataset are different from those in the paper, and other three experiments were similar to the data in the paper. My experimental result in the Inshop is about 89.9, but it is about 90.9 in the paper. I have been unable to find the reason. Is there anyone else who has the same problem as me?

**Comment by euwern** [comment]
Hey, there are few things that can result in the difference of results. For InShop, since the evaluation for training and testing is different, we find that using the same learning rate throughout the training is better than performing multistep learning rates.
You achieve this by running: "CUDA_VISIBLE_DEVICES=0,1 python train.py --dataset inshop  --config config/inshop.json --mode trainval --apex --seed 0" only without running the mode='train' first.  

Secondly, we also notice that the version of PyTorch could also cause the difference in the results. We ran our experiments in Pytorch V1.2.

**Comment by zcgit001** [comment]
Thanks for your reply.I found the problem about the learning rate yesterday, so I used the same learning rate instead of multistep learning rates to carry out the experiment on inshop yesterday and got the same result as in the paper.
Thank you very much again!
